src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+/* code here */
+@import "../utils/variables";
+
+.product {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  height: $card-height;
+
+  &:nth-child(-n + 4) {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-right: 48px;
+    margin-bottom: 55px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__photo {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  &__description {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+  &__code {
+    margin-bottom: 16px;
+    align-self: flex-start;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+  &__rating {
+    width: 166px;
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  &__rating-reviews {
+    justify-self: center;
+    align-self: center;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-main-accent;
+  }
+  &__cost {
+    width: 100%;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  &__cost-text {
+    font-weight: normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+    order: 0;
+  }
+  &__cost-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+    width: 166px;
```

Comment:
flex elements will take up all of the available space anyways, remove it
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/110053436/216615600-7d94f4e4-0f09-46b3-aab4-fde6fabc8b20.png)
i can't do it, because when i take off that property - width of link take own size (by content)